### ISSUE TITLE
1.1 < 1.1 return true when it should be false

### ISSUE THREAD

**Issue by regis-leray** [kypeli/sversion#1]
These current tests are failing

```
 "0.1.24 < 0.1.24" in {
      Version("0.1.24") < Version("0.1.24") mustEqual false
    }

    "1.1 < 1.1" in {
      Version("1.1") < Version("1.1") mustEqual false
    }
```

These current tests are success

```
 "0.1.24 > 0.1.24" in {
      Version("0.1.24") > Version("0.1.24") mustEqual false
    }

    "1.1 > 1.1" in {
      Version("1.1") > Version("1.1") mustEqual false
    }
```

**Comment by kypeli** [comment]
Thanks for reporting this. This has been fixed in commit 3555c35607108669e4756a0cbe1b2ea6f9e6ea23.

```
[info] Version < should
[info]   + 0.1.24 < 0.1.24 == false
```

At the same time, I added more tests to cover corner cases.

**Comment by regis-leray** [comment]
@kypeli  Thanks that was fast.
Are you planning to do a new release ?

**Comment by kypeli** [comment]
Yes, I just released version `1.1` to Sonatype repository so should be available shortly, after the repositories have synced. 

https://oss.sonatype.org/#nexus-search;quick~sversion

**Comment by regis-leray** [comment]
Thank so much
